### DISCUSSION TITLE
Travis Migration

### DISCUSSION THREAD

**Discussion by Noordfrees** [widelands/widelands#4912] [General]
Today I migrated the Travis builds from https://travis-ci.org/ to https://www.travis-ci.com/ due to their `.org` domain shutting down soonish. All new Travis builds are from now on built on `.com` instead of `.org`. For us everything *should* continue to work exactly like it did before. If you notice something wrong with the Travis integration (builds not triggered when they should be, check status not updated, etc) please mention it here.

**Comment by Noordfrees** [comment]
Two weeks ago Travis stopped building all new commits and PRs and it seems like nobody has noticed :P 

Apparently (this is poorly documented), we'd need to select a payment plan and request free open source credits and all that. If they want to discourage open source, let them – in my opinion our GitHub actions CI is way better than Travis anyway. 

Would anyone miss Travis integration? If there are no positive replies for a week I'll remove it.

**Reply by matthiakl** [reply]
> macOS 10.13 is neither supported by Apple nor by homebrew anymore. So I support removing it from the tests.

https://github.com/widelands/widelands/pull/4543#issuecomment-748633622

So I think we can safely drop it now. It only builds one setup, which even isn't officially supported anymore.
